README.md

```diff
@@ -1,56 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Covid19 Tracker Web app
 
+> A web app built with Covid-19-API that displays Covid 19 cases, deaths and recovery per country in the entire World
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JavaScript
+- React
+- Redux
+- WebPack 
+- Covid19 API
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo](https://634b18b7735def69fb36e1ca--musical-sunburst-e30cc3.netlify.app/)
 
 
-## Getting Started
+## Video Demo 
+
+[Video Demo](https://drive.google.com/file/d/15cvM6YNoK8lCaV5BIxsvKP2Tg24AxtFp/view?usp=sharing)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## ScreenShots
+### Desktop screen
+<img src="./Screenshots/Homepage1.png" alt="Rockets" width="500px"/>
+<img src="./Screenshots/Homepage2.png" alt="Missions" width="500px"/>
 
+### Mobile Screen
+<img src="./Screenshots/homepageMobile.png" alt="Profile" width="500px"/>
+<img src="./Screenshots/detailsMobile.png" alt="Profile" width="500px"/>
+
+
+## Getting Started
 
```

Comment:
- [ ] Kindly also add instructions in your ReadMe on how to _run the app, as well as test cases of your project, under the Getting Started section.